pkgs/development/python-modules/coffea/default.nix

```diff
@@ -31,20 +31,19 @@
 
 buildPythonPackage rec {
   pname = "coffea";
-  version = "2023.10.0.rc1";
+  version = "2023.12.0.rc1";
```

Comment:
2023.12.0 (not rc!) has just released
https://github.com/CoffeaTeam/coffea/releases/tag/v2023.12.0

Reply:
Indeed ! I just added it.